pandas/io/formats/css.py

```diff
@@ -27,12 +56,72 @@ def expand(self, prop, value: str):
     return expand
 
 
+def _border_expander(side: str = "") -> Callable:
+    """
+    Wrapper to expand 'border' property into border color, style, and width properties
+
+    Parameters
+    ----------
+    side : str
+        The border side to expand into properties
+
+    Returns
+    -------
+        function: Return to call when a 'border(-{side}): {value}' string is encountered
+    """
+    if side != "":
+        side = f"-{side}"
+
+    def expand(self, prop, value: str) -> tuple[str, str]:
```

Comment:
The `mypy` error you are seeing here might need `-> Generator`, instead of `tuple[str, str]`